extensions/resteasy-reactive/rest-client/deployment/src/test/java/io/quarkus/rest/client/reactive/proxy/HttpProxyDevServicesMultipleCustomProvidersTest.java

```diff
@@ -89,6 +90,7 @@ public void test() {
     }
 
     @Path("test")
+    @RegisterRestClient
```

Comment:
Why is this needed? AFAICT, the test was actually using programmatic building of the client

Reply:
> As far as I could tell, we don't really have a build item that provides us the list of Clients (without using the annotation).

You are probably right, but I think the ScanningResult has this information (or at least part of it). 

The reason I really need the programmatic stuff is that I use this way in Quarkus LangChain4j and from time to time I do use this proxy feature myself

Reply:
Ok, let me check that.

Reply:
🙏🏽